hazelcast/cluster.py

```diff
@@ -58,9 +68,11 @@ class ClusterService(object):
     """
 
     def __init__(self, internal_cluster_service):
+        # type: (Any) -> None
         self._service = internal_cluster_service
 
     def add_listener(self, member_added=None, member_removed=None, fire_for_existing=False):
+        # type: (Callable[[],Callable[[Any],List[Any]]], Callable[[],Callable[[Any],List[Any]]], bool) -> str
```

Comment:
`member_added` and `member_removed` are functions of the following type `(MemberInfo) -> None`, can you fix the types for them?